### PR TITLE
Context switch

### PR DESCRIPTION
**Pull request by javonmlee**
This pull request implements the C2 context switching feature.  A new dropdown has been added to the agents modal giving the option to change the C2 channel to either HTTP or GIST.

### CODE REVIEW

File: static/js/sections.js [hunk 1]
```diff
@@ -113,51 +113,67 @@ $(document).ready(function () {
...
                 targets: 5,
                 data: null,
+                render: function ( data, type, row, meta ){
+                    let str = "<select id=\""+data['paw']+"-c2\">";
+                    if ( data['c2'] == 'HTTP' ){
+                        str += "<option value=\"HTTP\" selected=\"selected\">HTTP</option>\n" +
+                               "<option value=\"GIST\">GIST</option>\n";
+                    } else {
+                        str += "<option value=\"HTTP\">HTTP</option>\n" +
+                               "<option value=\"GIST\" selected=\"selected\">GIST</option>";
```
Comment (wbooth) [inline static/js/sections.js]: could set two options and then select via js depending on data['c2'], this would shorten the code block

Comment (javonmlee) [reply static/js/sections.js]: How would you go about selecting the value base on data['c2'] in the scope of the render function using js?

Comment (wbooth) [reply static/js/sections.js]: it'd look something like 
`$("#select-id").val(data['c2']).select();`

Comment (javonmlee) [reply static/js/sections.js]: Would that need to be called outside of the function scope?

**Comment by javonmlee** [comment]
Updated in new PR

**Pull request closed without merging**